readme.md

```diff
@@ -160,6 +167,7 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Your Answer:** 
 
+I received an Unhandled promise rejection are depreceted error with the findOneAndUpdate(). I changed my function to (response => { res.json({ status, response })})
```

Comment:
Not sure if this is the right error here. My expectation would be that you would receive an error when running `.create().select()`.